email/testing.rst

```diff
@@ -67,4 +63,16 @@ to get information about the messages sent on the previous request::
         }
     }
 
+Troubleshooting
+---------------
+
+The email collector is only available when the profiler is enabled and collects
+information, as explained in :doc:`/testing/profiling`.
+
+If the action that sends the email redirects to another page (for example when
+you send an email after a form is processed and before redirecting to another
+page), make sure that the test client doesn't follow the redirects, as explained
+in :doc:`/testing`. Otherwise, the collector will lose the original email when
```

Comment:
I think the last sentence is a bit misleading. It's not that the collector loses the mail, but that you are just looking at the "wrong" profile after being redirected, right?